index.html

```diff
@@ -172,9 +172,9 @@ <h2 class="multi-post">
 
                 
 
-                <div class="button project-button">
+                <button class="button project-button" type="button">
                     See Project
-                </div>
+                </button>
 
             </div>
```

Comment:
- [ ] The work section project cards suppose to be generated dynamically using javascript not through HTML code So please remove the work section HTML code and implement it dynamically using javascript.

https://github.com/udeaghad/Create-mobile-version/blob/2fc91d063a682d2876e1f03e6f499223b799dce2/index.html#L135-L385

![image](https://user-images.githubusercontent.com/68796232/168076268-f5d57796-7b55-47db-9ee2-25b56a579c9e.png)